utils/utils.go

```diff
@@ -226,6 +226,14 @@ func IsAddressLegal(address string) bool {
 	return isLegal
 }
 
+// GetEnv get value from environment variable.
+func GetEnv(envName string, defaultValue string) string {
+	if val := os.Getenv(envName); val != "" {
```

Comment:
If an environment variable is actually set as an empty string, this will return a default value. It will be more robust to use `os.LookupEnv` ([docs](https://pkg.go.dev/os#LookupEnv)) that returns the value, and a bool to indicate if the variable is set.

Reply:
@1046102779

Reply:
@shubham1172 fixed. Thanks.